terraform/clusters/davidpye.re-run-sandbox.aws.ext.govsvc.uk/kubeconfig

```diff
@@ -4,7 +4,7 @@ clusters:
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURGekNDQWYrZ0F3SUJBZ0lSQUl2SWNZNy9WejFSbnJjUm9jTittemN3RFFZSktvWklodmNOQVFFTEJRQXcKSlRFUk1BOEdBMVVFQ2hNSVltOXZkR3QxWW1VeEVEQU9CZ05WQkFNVEIydDFZbVV0WTJFd0hoY05NVGd4TWpBMwpNVFEwTVRFNFdoY05NVGt4TWpBM01UUTBNVEU0V2pBbE1SRXdEd1lEVlFRS0V3aGliMjkwYTNWaVpURVFNQTRHCkExVUVBeE1IYTNWaVpTMWpZVENDQVNJd0RRWUpLb1pJaHZjTkFRRUJCUUFEZ2dFUEFEQ0NBUW9DZ2dFQkFMY3UKWTBTZ3loVDBNbkZKdlM2MFVmMTF0dktOams2R1FodkpTQnpTV0kvVE1BbFJ4SjlVY3QrUDFsaW9TcXhlKzFlTQpXTmhTdHNFT3hZVXMwUmFXQ0owZ3FicFdublpvNks0dlZDUXR6eTNkYU5mM24wQkNKejFwckN6VXlMYkZTQTRRCm90V25kTXJMSzhsSEpBVURHUW4zQUhBL29UdXQ0SWVNaDM4RzFJNkxjeDVuc1VkblpHcjJtZVRFTnhkeFhiY0sKRWpRcVZBM0daNUpKYjFnWkVOT0J1bDdyU0NVNWp1NWJ6OW1UcXcxOW1hbzNOaEowakxZTUlPU0cyQUpyTmJYZwpKTGFDQU00alNJeDRib2pZV2dWblBWQzkrZXZMK0p4SmRjNW9LNXVuK0sraVNmQjJER0lqLzI0ZktwU2lZV2tICnJIME5YdEF3RDgrZEY2VHFWZjBDQXdFQUFhTkNNRUF3RGdZRFZSMFBBUUgvQkFRREFnS2tNQThHQTFVZEV3RUIKL3dRRk1BTUJBZjh3SFFZRFZSME9CQllFRkcrelJBOHI0UVN1SHMwbWlpWmNwcUl4VVF2Y01BMEdDU3FHU0liMwpEUUVCQ3dVQUE0SUJBUUNwVGRWNnNZWjNTekczV0JUSkx0L1pwMGFIN2pwNWYraXB4TWZ3QnNEZDZMWmVqYllmCjNZZjdzV3ZvUzVWS2JxUVpCeVhDQU5mUm9SMCtZSXhMVVZselRQMWFyTWJ0L0JYMEZWTlFTVGN4bHNNTlhMSGwKSXc2Z3YzSnZpU1R3S0w2NlFYdHJZQ3ZZWXMvLzVtSHV4Znp2V1ZpVHl6bmdKcGx5MUJxTmkwU1dneFBTR0xtSwpQeEdKZDBpbDVuZ0ZiN2QrUVd3ZkwyZEFPRFFzS2JITE9pWGNIUXZYMjJYa25sZXZsSm1UUjhnTXNLTC9MTEQwCmpZdjM1Z3ZXT3JvTmwxQ3BNZ3VJWHArck1lNFQrdEc4MXZiYW9weERoVjJBTlRJQU8vSVVVWFI5K3pmTkVjQXAKWWtzRWd4emJtcnl6eTZnZ3ZiZlgwWlBVa084OTFiSjRyenU4Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
```

Comment:
Can we get rid of this file?

It's a pain to keep it up to date and should always be available through `terraform output`.

Reply:
yes of course.